project.py

```diff
@@ -7,4 +7,19 @@
 current_time = datetime.datetime.now()
 
 # Printing value of now.
-print("Time now at greenwich meridian is:", current_time)
\ No newline at end of file
+print("Time now at greenwich meridian is:", current_time)
+
+#Printing out the distance between 19/07/23 and 01/01/24 
+from datetime import date
+f_date = date(2023, 7, 19)
+l_date = date(2024, 1, 1)
+delta = l_date - f_date
+print(delta.days)
+
+#Create a list of birthdays in an array with names 
+ 
+from datetime import date
```

Comment:
it appears that we are importing the same package twice in this file, I'm pretty sure we don't need to do this.

I also think it's convention in python to put all imports at the top of the file (like go)

Maybe we can move the import to the top of the file, and remove the duplicate?